x-pack/plugins/lens/public/plugin.ts

```diff
@@ -234,6 +235,7 @@ export class LensPlugin {
   }
 
   start(core: CoreStart, startDependencies: LensPluginStartDependencies): LensPublicStart {
+    setStartServices(core, startDependencies);
```

Comment:
We don't use this pattern any more, and I'll leave another comment pointing you to the new pattern.